libs/providers/aws-ssm/src/e2e/e2e.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { OpenFeature } from '@openfeature/server-sdk';
```

Comment:
One more thing here... could you either run this as an `e2e` target or, rename this to `integration` or something like that?

I'd prefer to keep things consistent and run all tests called `e2e` in the `e2e` target, which takes longer since in some cases containers are run.